src/algebra/group/ulift.lean

```diff
@@ -0,0 +1,92 @@
+/-
+Copyright (c) 2020 Scott Morrison. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Scott Morrison
+-/
+import data.equiv.mul_add
+
+/-!
+# `ulift` instances for groups and monoids
+
+This file defines instances for group, monoid, semigroup and related structures on `ulift` types.
```

Comment:
Could you add a sentence on what `ulift` is to these module docs? Someone stumbling across these files / doc pages might not know and it's not something that's in TPIL.

Reply:
Sure!